web/src/routes/(user)/people/[personId]/+page.svelte

```diff
@@ -462,7 +473,7 @@
                   <div
                     class="flex {index === suggestedPeople.length - 1
                       ? 'rounded-b-lg'
-                      : 'border-b dark:border-immich-dark-gray'} place-items-center bg-gray-100 p-2 dark:bg-gray-700"
+                      : ''} border-t dark:border-immich-dark-gray place-items-center bg-gray-100 p-2 dark:bg-gray-700"
```

Comment:
Is this on purpose? You now always add the `border-t ...` classes regardless of the length.

Reply:
OK, sounds good. Maybe you can move these "always" classes to the beginning of the list instead then. (and the ternary stuff could instead be at the end.